src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,100 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final float LOAD_FACTOR = 0.75f;
+    private static final int COEFFICIENT_GROW = 2;
+    private static final int DEFAULT_CAPACITY = 16;
+    private Node<K, V>[] map;
+    private int capacity;
+    private int size;
+
+    public MyHashMap() {
+        capacity = DEFAULT_CAPACITY;
+        map = new Node[capacity];
+    }
 
     @Override
     public void put(K key, V value) {
-
+        if (size + 1 >= capacity * LOAD_FACTOR) {
+            resizeMap();
+        }
+        putItem(map, capacity, key, value);
     }
 
     @Override
     public V getValue(K key) {
+        int index = getBucketIndex(key);
+        Node<K, V> node = map[index];
+        while (node != null) {
+            if (node.key == key || (key != null && key.equals(node.key))) {
```

Comment:
we can already use Objects.equals(), so let's be consistent and use it here as well

Reply:
ok